yb-voyager/cmd/exportData.go

```diff
@@ -148,6 +148,7 @@ func exportDataOffline() bool {
 			utils.ErrExit("Exiting at user's request. Use `--exclude-table-list` flag to continue without these tables")
 		}
 	}
+	finalTableList = filterTableWithEmptySupportedColumnList(finalTableList, tablesColumnList)
```

Comment:
Slightly unrelated question: What happens if as a result of all the filtering that we do (empty tables, unsupported columns, etc), we end up with no tables. Do we throw an error?

Reply:
Oh yes, thanks we do that after filtering of tablelist for other cases few lines above, but I missed that here. Will now do that after this case of filtering. Thanks

Reply:
yes, we return from there by saying no tables to be exported and hence, export is completed.

Reply:
Got it, thanks!